src/puppetlabs/puppetdb/command.clj

```diff
@@ -233,9 +233,9 @@
 
 (defmacro upon-error-throw-fatality
   [& body]
-  `(try+
+  `(try
     ~@body
-    (catch Throwable e#
+    (catch Exception e#
```

Comment:
So what you're saying is that `(catch Throwable` is the right thing to do here?

:duck: